README.md

```diff
@@ -41,15 +41,19 @@ $Env:Path += ";$(pwd)"
 
 ## Getting the Code and Building Electron
 
-After installing build-tools, you can run a new Electron build with this command:
+You can run a new Electron build with this command:
 
 ```sh
 # The 'Hello, World!' of build-tools: get and build `master`
 
 # On macOS if you aren't sure you have the right SDK version you should run
 e load-macos-sdks
 
-e init --root=/path/to/new/electron/directory --bootstrap testing
+# Choose the directory where Electron's source and build files will reside.
+# You can specify any path you like; this command defaults to ~/projects/electron.
+# If you're going to use multiple branches, you may want something like:
+# * `--root=~/electron-gn/branch` (e.g. `~/electron-gn/master`)
+e init --root=~/projects/electron --bootstrap testing
```

Comment:
I'd suggest we use consistent directories here instead of mixing `~/electron-gn` and `~/projects/electron` as the base.  Maybe something like

```text
# If you're going to use multiple branches, you may want something like
# `--root=~/electron-gn/branch` (e.g. `~/electron-gn/master`)
e init --root=~/electron-gn --bootstrap testing
```